conf/app.routes

```diff
@@ -4,6 +4,8 @@ GET         /assets/*file                                    controllers.Assets.
 
 GET        /                                                 controllers.SoftwareChoicesController.show
 POST       /                                                 controllers.SoftwareChoicesController.search
+
+GET        /software-information/                            controllers.SoftwareInformationController.softwareInformation(providerName: String)
```

Comment:
remove the last / in /software-information/ or make it /software-information/:providerName
first approach makes the url /software-information?providerName=Cirrostratus Exedra
second /software-information/Cirrostratus Exedra
I believe we normally only use the second if it's not the last part of the url